src/storage/txn/scheduler.rs

```diff
@@ -1101,11 +1094,19 @@ impl Scheduler {
         let tag = cmd.tag();
         if readcmd {
             worker_pool.execute(move |ctx: &mut SchedContext| {
+                let _processing_read_timer = SCHED_PROCESSING_READ_HISTOGRAM_VEC
```

Comment:
@overvenus we can supply a LocalHistogramVec

Reply:
It's now available. https://github.com/pingcap/rust-prometheus/pull/129